src/Workspaces/Core/Portable/Rename/HeavyweightRenameLocations.ReferenceProcessing.cs

```diff
@@ -14,111 +16,17 @@
 using Microsoft.CodeAnalysis.Shared.Extensions;
 using Microsoft.CodeAnalysis.Text;
 using Roslyn.Utilities;
-using System.Text.RegularExpressions;
-using System.Collections.Immutable;
-using System.Text;
 
 namespace Microsoft.CodeAnalysis.Rename
 {
     /// <summary>
     /// A helper class that contains some of the methods and filters that must be used when
     /// processing the raw results from the FindReferences API.
     /// </summary>
-    internal sealed partial class RenameLocations
+    internal sealed partial class HeavyweightRenameLocations
     {
         internal static class ReferenceProcessing
         {
-            /// <summary>
-            /// Given a symbol in a document, returns the "right" symbol that should be renamed in
-            /// the case the name binds to things like aliases _and_ the underlying type at once.
-            /// </summary>
-            public static async Task<ISymbol?> TryGetRenamableSymbolAsync(
-                Document document, int position, CancellationToken cancellationToken)
```

Comment:
all these nested static methods were used in many locations in roslyn.  They moved to RenameUtilities as it really didn't make sense to update them to say HeavyweightRenameLocations.ReferenceProcessing.DoSomething.  The helpers have nothing to do with HeavyweightRenameLocations at all.